generator.yml

```diff
@@ -178,6 +189,9 @@ targets:
   - &default     {targets: [default]}
   - &kernel      {targets: [kernel]}
   - &kernel_dtbs {targets: [kernel,dtbs]}
+chromeos_configs:
+  - &arm64-cros-configs  {config: [chromeos/config/chromeos/base.config, chromeos/config/chromeos/arm64/common.config, chromeos/config/chromeos/arm64/chromiumos-arm64.flavour.config]}
+  - &x86_64-cros-configs {config: [chromeos/config/chromeos/base.config, chromeos/config/chromeos/x86_64/common.config, chromeos/config/chromeos/x86_64/chromiumos-x86_64.flavour.config]}
```

Comment:
Is this something tuxbuild supports that we simply haven't used before (using multiple config files?)

Reply:
Yes, although using a config fragment as the first ("base") configuration is new: https://gitlab.com/Linaro/tuxmake/-/merge_requests/269